### PR TITLE
(ClipboardFusion) Moving to core packages, along with an update script

### PR DESCRIPTION
**Pull request by The-Running-Dev**

### CODE REVIEW

File: automatic/clipboardfusion/clipboardfusion.nuspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ClipboardFusion makes it easy to remove clipboard text formatting,
+    replace clipboard text or run powerful macros on your clipboard contents.
+    You can even sync your clipboard with other computers and mobile devices.
+    </description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>clipboardfusion clipboard sync history scrubbing macros</tags>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatci/clipboardfusion</packageSourceUrl>
+    <owners>chocolatey,ferventcoder</owners>
```
Comment (pascalberger) [inline automatic/clipboardfusion/clipboardfusion.nuspec]: Why ferventcoder? The package was maintained until now by keithlammers

Comment (pascalberger) [inline automatic/clipboardfusion/clipboardfusion.nuspec]: Add releaseNotes tag (https://www.clipboardfusion.com/ChangeLog/)

File: automatic/clipboardfusion/clipboardfusion.nuspec [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <licenseUrl>https://www.clipboardfusion.com/License/</licenseUrl>
+    <projectUrl>https://www.clipboardfusion.com</projectUrl>
+    <iconUrl>https://images.binaryfortress.com/ChocolateyIcons/ClipboardFusion.png</iconUrl>
+    <docsUrl>https://www.clipboardfusion.com/FAQ/</docsUrl>
+    <mailingListUrl>https://www.clipboardfusion.com/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ClipboardFusion makes it easy to remove clipboard text formatting,
+    replace clipboard text or run powerful macros on your clipboard contents.
+    You can even sync your clipboard with other computers and mobile devices.
```
Comment (pascalberger) [inline automatic/clipboardfusion/clipboardfusion.nuspec]: Add a notes heading as per [§.1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description) with a paragraph mentioning that the free license is only for personal & non-commercial use

File: automatic/clipboardfusion/update.ps1 [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+$global:getBetaVersion = $false
+
+function global:au_BeforeUpdate {
+  $Latest.ChecksumType32 = 'sha256'
+
+  Get-RemoteFiles -Purge
+
+  $file = Get-ChildItem $PSScriptRoot\tools -Filter "*.exe" | select -First 1 | % { Get-Item $_.FullName }
+  Remove-Item $file -Force
```
Comment (pascalberger) [inline automatic/clipboardfusion/update.ps1]: Add `-ErrorAction SilentlyContinue`

File: automatic/clipboardfusion/tools/chocolateyInstall.ps1 [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿$downloadDir        = Get-PackageCacheLocation
```
Comment (pascalberger) [inline automatic/clipboardfusion/tools/chocolateyInstall.ps1]: Add `$ErrorActionPreference = 'Stop'` at the top

File: automatic/clipboardfusion/clipboardfusion.nuspec [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    </description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>clipboardfusion clipboard sync history scrubbing macros</tags>
```
Comment (AdmiringWorm) [inline automatic/clipboardfusion/clipboardfusion.nuspec]: if this is both a trial and a freeware, both tags should be used.

**Comment by The-Running-Dev** [comment]
This should address the PR changes requested.

**Comment by The-Running-Dev** [comment]
I already committed all the requested changes. Was there something else I was missing?

**Comment by pascalberger** [comment]
@The-Running-Dev Thanks LGTM

**Comment by pascalberger** [comment]
@The-Running-Dev Were the original maintainers contacted that this packages is moved to the core team repository and that they need to remove any automation they have setup?

**Comment by AdmiringWorm** [comment]
@The-Running-Dev looks like this package failed the automated tests on chocolatey.org.
Mind having a look at it? [Log](https://gist.github.com/choco-bot/b1f5aeb46a22a892b56a45a4c2bfb9c2)
My first guess is because of the use of the `file` and `fileType` variables (you don't need those BTW)

**Comment by AdmiringWorm** [comment]
@pascalberger He mentioned in a previous comment that the original maintainer was on board with it being moved to the core repo.

**Pull request merged**